CRM/Utils/Hook.php

```diff
@@ -1759,6 +1759,23 @@ public static function alterTemplateFile($formName, &$form, $context, &$tplName)
     );
   }
 
+  /**
+   * Initialize the cryptographic service.
```

Comment:
Not sure this comment is exactly right

Reply:
For me, it's descriptive. The hook literally runs via `createDefaultRegistry()` when initializing the `CryptoRegistry` service. But if something else describes it better, that's cool.

Reply:
I guess to me we have initialized it already because we are passing through the class object to the hook, Maybe something about modifying the cryptographic service?

Reply:
Just to close this subthread - the text was update (i.e. "Initialize the cryptographic service" ==> "Register cryptographic resources, such as keys and cipher-suites.")